tests/lightning_tensor/test_var.py

```diff
@@ -0,0 +1,407 @@
+        )
+
+        tape = qml.tape.QuantumScript(
+            [qml.RX(phi, wires=[0]), qml.RX(-1.1 * phi, wires=[0])],
+            [qml.var(obs)],
+        )
+
+        calculated_val = execute(dev, tape)
+        reference_val = calculate_reference(tape)
+
+        tol = 1e-5 if dev.c_dtype == np.complex64 else 1e-7
+
+        assert np.allclose(calculated_val, reference_val, tol)
+
+
+@pytest.mark.parametrize("theta, phi, varphi", list(zip(THETA, PHI, VARPHI)))
+class TestTensorVar:
+    """Test tensor variances"""
+
+    def test_PauliX_PauliY(self, theta, phi, varphi, dev, tol):
+        """Tests a tensor product involving PauliX and PauliY."""
+
+        with qml.tape.QuantumTape() as tape:
+            qml.RX(theta, wires=[0])
+            qml.RX(phi, wires=[1])
+            qml.RX(varphi, wires=[2])
+            qml.CNOT(wires=[0, 1])
+            qml.CNOT(wires=[1, 2])
+            qml.var(qml.PauliX(0) @ qml.PauliY(2))
+
+        calculated_val = execute(dev, tape)
+        reference_val = calculate_reference(tape)
+
+        tol = 1e-5 if dev.c_dtype == np.complex64 else 1e-7
+
+        assert np.allclose(calculated_val, reference_val, tol)
+
+    def test_PauliZ_identity(self, theta, phi, varphi, dev, tol):
+        """Tests a tensor product involving PauliZ and Identity."""
+
+        with qml.tape.QuantumTape() as tape:
+            qml.Identity(wires=[0])
+            qml.RX(theta, wires=[0])
+            qml.RX(phi, wires=[1])
+            qml.RX(varphi, wires=[2])
+            qml.CNOT(wires=[0, 1])
+            qml.CNOT(wires=[1, 2])
+            qml.var(qml.PauliZ(0) @ qml.Identity(1) @ qml.PauliZ(2))
+
+        calculated_val = execute(dev, tape)
+        reference_val = calculate_reference(tape)
+
+        tol = 1e-5 if dev.c_dtype == np.complex64 else 1e-7
+
+        assert np.allclose(calculated_val, reference_val, tol)
+
+    def test_PauliZ_hadamard_PauliY(self, theta, phi, varphi, dev, tol):
+        """Tests a tensor product involving PauliY, PauliZ and Hadamard."""
+
+        with qml.tape.QuantumTape() as tape:
+            qml.RX(theta, wires=[0])
+            qml.RX(phi, wires=[1])
+            qml.RX(varphi, wires=[2])
+            qml.CNOT(wires=[0, 1])
+            qml.CNOT(wires=[1, 2])
+            qml.var(qml.PauliZ(0) @ qml.Hadamard(1) @ qml.PauliY(2))
+
+        calculated_val = execute(dev, tape)
+        reference_val = calculate_reference(tape)
+
+        tol = 1e-5 if dev.c_dtype == np.complex64 else 1e-7
+
+        assert np.allclose(calculated_val, reference_val, tol)
+
+
+# Define the parameter values
+THETA = np.linspace(0.11, 1, 3)
+PHI = np.linspace(0.32, 1, 3)
+
+
+@pytest.mark.parametrize("theta, phi", list(zip(THETA, PHI)))
+def test_multi_qubit_gates(theta, phi, dev, tol):
+    """Tests a simple circuit with multi-qubit gates."""
+
+    ops = [
+        qml.PauliX(wires=[0]),
+        qml.RX(theta, wires=[0]),
+        qml.CSWAP(wires=[7, 0, 5]),
```

Comment:
Nice one!

Reply:
This is an example of multi-qubit gates targeting non-adjacent wires. With the new features in `quimb 1.8.1` this should not cause any problem